package.json

```diff
@@ -6,19 +6,20 @@
     "url": "https://github.com/momentum-design/momentum-ui.git"
   },
   "scripts": {
-    "bootstrap": "yarn build:tokens && lerna bootstrap && lerna link && cd ./web-components && yarn",
+    "bootstrap": "lerna bootstrap && lerna link",
     "bootstrap:clean": "run-s clean bootstrap",
     "build": "yarn build:all",
-    "build:all": "run-s build:icons build:core build:react build:charts",
+    "build:all": "yarn build:icons && yarn build:core && yarn build:utils && yarn build:tokens && yarn build:react && yarn build:web-components && yarn build:charts",
```

Comment:
Why the removal of `run-s`?

Reply:
@pauljeter it was causing Cirvcle CI build errors, because 'run-s' is not defined in that runtime

Reply:
@pauljeter  is there a way to add that module to Circle CI to use?